shared/actions/wallets.js

```diff
@@ -41,12 +44,64 @@ const loadPaymentsSuccess = (res: RPCTypes.PaymentsPageLocal, action: WalletsGen
   )
 }
 
+const linkExistingAccount = (state: TypedState, action: WalletsGen.LinkExistingAccountPayload) => {
+  const {name, secretKey} = action.payload
+  return RPCTypes.localLinkNewWalletAccountLocalRpcPromise(
+    {
+      name,
+      secretKey: secretKey.stringValue(),
+    },
+    Constants.linkExistingWaitingKey
+  )
+    .then(accountID => [
+      WalletsGen.createLoadAccounts(),
+      WalletsGen.createSelectAccount({accountID: Types.stringToAccountID(accountID), show: true}),
```

Comment:
also this is the pattern we want to avoid. When this thing is done it should emit some kind of action, and as a side effect these things should happen, or maybe one happens, and as a side effect the other things happen etc.
createLinkExistingAccountError shouldn't be necessary as the reducer for the action that is emitted by this should handle it in the reducer